src/util/util.c

```diff
@@ -191,6 +191,8 @@ char *joinpath(const char * path1, const char * path2_in) {
         ABORT(255);
```

Comment:
Should the free also go in this if statement above?

Reply:
I didn't think it was worth doing since the program dies there anyway.

Reply:
Your right.